Makefile

```diff
@@ -700,6 +702,7 @@ edpm_deploy_generate_keys:
 	$(eval $(call vars,$@,dataplane))
 	devsetup/scripts/gen-ansibleee-ssh-key.sh
 	bash scripts/gen-edpm-nova-migration-ssh-key.sh
+        bash scripts/gen-edpm-bm-root-password-secret.sh
```

Comment:
Run this script only if $BM_ROOT_PASSWORD has been set.

Reply:
> Run this script only if $BM_ROOT_PASSWORD has been set.

ack

Reply:
> > Run this script only if $BM_ROOT_PASSWORD has been set.
> 
> ack

done